### PR TITLE
9118 news page layout

### PR DESCRIPTION
**Pull request by margOnline**
[TP 9118](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5580782241563718174&appConfig=eyJhY2lkIjoiRTQ4MDEyNENDOUYxRDU4RDg0RTEyRjNCODBEN0VBMUYifQ==&searchPopup=userstory/9118)

There is a fincap requirement for editors to be able to add news pages. These pages are similar to articles but include a date which enables the editors to fine tune the order in which news items are displayed on the Latest News page. [Related TP story, 9117](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5580782241563718174&appConfig=eyJhY2lkIjoiRTQ4MDEyNENDOUYxRDU4RDg0RTEyRjNCODBEN0VBMUYifQ==&searchPopup=userstory/9117)

### CODE REVIEW

File: db/seeds/fincap.seeds.rb [hunk 1]
```diff
@@ -481,6 +493,8 @@
 tag = Tag.create(
   value: 'how-can-we-improve-the-financial-capability-of-young-adults'
 )
+mobile_payments_tag = Tag.create(value: 'mobile-payments')
+secure_payments_tag = Tag.create(value: 'secure-payments')
```
Comment (tomas-stefano) [inline db/seeds/fincap.seeds.rb]: Is there a reason for the tags?

**Comment by willhall88** [comment]
is this correct to be merging into `feature/8847-fincap-migration` branch or should it be going into `fincap`?

**Comment by margOnline** [comment]
good spot @willhall88

File: db/seeds/fincap.seeds.rb (hunk 1 above)
Comment (margOnline) [reply db/seeds/fincap.seeds.rb]: @tomas-stefano  To ensure they appeared on the page. I will also need them for the Latest News Component. A bit premature and I can remove them if you want

**Pull request merged**